src/main/java/com/example/hangman/HelloController.java

```diff
@@ -0,0 +1,76 @@
+package com.example.hangman;
+
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.net.URL;
+import java.util.ResourceBundle;
+
+public class HelloController implements Initializable {
+    @FXML
+    private Label answer;
+    private String random;
+    @FXML
+    private ImageView image;
+    private int count = 0;
```

Comment:
**Can be simplified:**
```suggestion
    private int count;
```
<hr />

all **fields** default to some values.
e.g.
### Primitives
`byte`, `short`, `int`, `long`, `float`, `double` default to `0`
`boolean` to `false`
`char` to `'\u0000'`
### non primitives (referenced types)
all of them default to `null`